ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.10__okd.yaml

```diff
@@ -59,7 +59,18 @@ tests:
 - as: aws-e2e-operator
   steps:
     cluster_profile: aws
+    env:
+      COMMUNITY: "true"
     test:
+    - as: test
+      commands: curl -L https://github.com/mikefarah/yq/releases/download/v4.13.5/yq_linux_amd64
+        -o /tmp/yq && chmod +x /tmp/yq && git clone https://github.com/openshift/windows-machine-config-operator.git
+        && cd windows-machine-config-operator && make community-bundle
```

Comment:
```suggestion
        commands: |-
          #!/bin/bash
          curl -L https://github.com/mikefarah/yq/releases/download/v4.13.5/yq_linux_amd64 -o /tmp/yq 
          chmod +x /tmp/yq
          git clone https://github.com/openshift/windows-machine-config-operator.git
          cd windows-machine-config-operator
          make community-bundle
```

Reply:
You could also add an inline comment within this script explaining why yq is needed

Reply:
Thanks, added this suggestion and the comment